roombasim/pittras/config.py

```diff
@@ -60,6 +61,12 @@
 # timeout for transforms
 PITTRAS_TAKEOFF_TRANSFORM_TIMEOUT = 0.2
 
+# velocity with which to land with
+PITTRAS_LAND_VELOCITY = -0.2
+
+# height below which landing is considered finished
+PITTRAS_LAND_HEIGHT_TOLERANCE = 0.15
```

Comment:
This parameter isn't actually used on the drone, RoombaSIM should probably just check if the drone is going to move into or below the floor and check if the landing is complete that way.  Then when the landing is complete it should probably set the height to 0 exactly.

Reply:
Hmm, should I check if the `height` is equal to zero in the task to determine if the task has completed?

Reply:
Yes, that will work.  The current implementation in `agent/drone.py` sets the height to exactly `0` when you hit the floor, but I'm not sure if you want to make that the defined behavior or not.  That's a question for @hgarrereyn